fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tags/v1/CommonEnchantmentTags.java

```diff
@@ -0,0 +1,25 @@
+package net.fabricmc.fabric.api.tags.v1;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.v1.TagRegistration;
+
+public class CommonEnchantmentTags {
+	/**
+	 * A tag containing enchantments that increase or other improve drops from blocks.
```

Comment:
What would improvement include? Would Silk Touch be part of this tag because it 'improves' the loot of blocks that otherwise don't drop anything?

Reply:
I wouldn't include ST in this tag, I will add clarification on what improvement is.